tests/lazy-loading.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { test, expect } from '@playwright/test';
+
+test('does not load heavy dependencies by default', async ({ page }) => {
```

Comment:
Is there a way to verify the requests do actually happen once they are required?

Reply:
Added a check for this and included rough size checks to ensure the chunk is not just some 1kB wrapper. Although the size checks don't seem to work in safari (size listed as 0), but the request still completes